### PR TITLE
Add tests for agent framework traces components

### PR DESCRIPTION
**Pull request by gaobinlong**
### Description

Add unit tests for agent framework traces components: `agent_framework_traces` and `agent_framework_traces_flyout_body`.

### CODE REVIEW

**Review by SuZhou-Joe — APPROVED** [review]

**Comment by Hailong-am** [comment]
I have update content of trace step in PR #59, that may impact the snapshot. you can get the latest code and update snapshot accordingly.

**Review by Hailong-am — APPROVED** [review]

File: public/components/agent_framework_traces_flyout_body.test.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+  it('not back button if traceId does not exist', async () => {
+    jest.spyOn(chatContextExports, 'useChatContext').mockReturnValue({
+      traceId: undefined,
+    });
+    render(<AgentFrameworkTracesFlyoutBody />);
+    expect(screen.queryAllByLabelText('back')).toHaveLength(0);
+  });
+
+  it('fullscreen with history open', async () => {
```
Comment (wanglam) [inline public/components/agent_framework_traces_flyout_body.test.tsx:43]: Shall we update the test description? Seems the history is not open here. We may need to add a separate test for history open.

Comment (gaobinlong) [reply public/components/agent_framework_traces_flyout_body.test.tsx]: Thanks, I've added a new test for it.

File: public/components/agent_framework_traces_flyout_body.test.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    expect(screen.queryAllByLabelText('back')).toHaveLength(1);
+    act(() => {
+      fireEvent.click(screen.getByText('Back'));
+    });
+    await waitFor(() => {
+      expect(onCloseMock).toHaveBeenCalledWith('chat');
+    });
+  });
+
+  it('not back button if traceId does not exist', async () => {
```
Comment (ruanyl) [inline public/components/agent_framework_traces_flyout_body.test.tsx]:
```suggestion
  it('should not display back button if traceId does not exist', async () => {
```

**Review by ruanyl — APPROVED** [review]

File: public/components/agent_framework_traces_flyout_body.test.tsx (hunk 2 above)
Comment (gaobinlong) [reply public/components/agent_framework_traces_flyout_body.test.tsx]: Thanks, I've changed that.

**Pull request merged**